### DISCUSSION TITLE
Mastodon embed timeline / iphone posts

### DISCUSSION THREAD

**Discussion by cosinebob** [mastodon/mastodon#22638] [Q&A]
Reference is: https://gitlab.com/idotj/mastodon-embed-feed-timeline

Got this working just fine BUT, when I post from iphone the post does not show in the timeline.  It shows on the mastodon app on the iphone where I am logged in.  It shows on desktop where I am logged in.  It does NOT show on desktop where I am NOT logged in.   There must be some setting on the iphone causing these posts to be private?  
I looked but cannot find anywhere to configure that, either on the iphone or on the desktop.
Suggestions appreciated.

**Comment by idotj** [comment]
Hi @cosinebob I found your question doing a random search about my widget developed in gitlab.
The embed feed timeline is a development aside of the official Mastodon app.

I will suggest first to try the last version available and check again if you can reproduce the same problem.
Feel free to open an issue in case you can't see your post in the embed timeline widget.

https://gitlab.com/idotj/mastodon-embed-timeline/-/issues
